pkg/monitortests/node/legacynodemonitortests/exclusions.go

```diff
@@ -47,6 +47,13 @@ func isThisContainerRestartExcluded(locator string, exclusion Exclusion) bool {
 		{
 			containerName: "container/ovn-acl-logging", // https://issues.redhat.com/browse/OCPBUGS-42344
 		},
+		{
+			containerName: "container/managed-upgrade-operator", // https://issues.redhat.com/browse/OSD-26270
```

Comment:
Oof. How?  Won't block merge, but I really hope this stabilizes

Reply:
Yea I'm chasing them for fixes for all these, but trying to get the job to pass so I can try to hold on to what I've got working